src/Classes/DKP_Entry.lua

```diff
@@ -143,50 +143,42 @@ function DKP_Entry:GetFormattedNames()
         end
     end
 
-    local formattedNames = ''
-
-    local function compare(a,b)
-        local a_member = Guild:GetMemberByName(a)
-        local b_member = Guild:GetMemberByName(b)
-
-        if a_member == nil then
-            return true
-        elseif b_member == nil then
-            return false
-        end
```

Comment:
This section was causing the invalid order function for sorting issue. In the case where both a and b are nil because they don't belong to the guild, this isn't deterministic. compare(a,b) returns a < b while compare(b,a) returns b > a, which can't happen. I avoided this case entirely by looking up members and removing nil values ahead of time.